lib/watch/index.js

```diff
@@ -26,7 +26,6 @@ const stop = (watcher = null, err = null) => {
   childs = []
   if (err) { console.log(chalk.red(err)) }
   if (watcher) {
-    allStop = true
     watcher.close()
```

Comment:
Why you delete `allStop = true` here?

Reply:
My bad, I don't know where it comes from